docs/remote.md

```diff
@@ -49,6 +49,16 @@ When loaded, a remote driver process receives an HTTP POST on the URL `/Plugin.A
 
 Other entries in the list value are allowed; `"NetworkDriver"` indicates that the plugin should be registered with LibNetwork as a driver.
 
+### Set capability
+
+After Handshake, the remote driver will receive another POST message to the URL `/NetworkDriver.GetCapability` with no payload. The driver's response should have the form:
+
+{
+	"Scope": "local"
+}
+
+Value of "Scope" should be either "local" or "global" which indicates the capability of remote driver, other values beyond these will make driver's registration failed that remote driver won't work.
```

Comment:
minor:
s / `other values beyond these will make driver's registration failed that remote driver won't work.` / `values beyond these will fail driver's registration and return an error to the caller.`